ci_framework/roles/openshift_provisioner_node/vars/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# While options found within the vars/ path can be overridden using extra
+# vars, items within this path are considered part of the role and not
+# intended to be modified.
+
+# All variables within this role should have a prefix of "cifmw_opn"
+
+cifmw_opn_prov_bridge_ipv4: "172.22.0.254/24"
+cifmw_opn_user: "kni"
+cifmw_opn_required_packages:
+  - python3-devel
+  - ipmitool
```

Comment:
Why we need IPMI cli tool? Haven't spotted were are we using it. Same for jq.

Reply:
I had taken it from an earlier investigation. The below two packages are also not required.